sample-kotlin/src/test/java/com/infullmobile/android/infullmvp/sample/sample_mvp_custom_view/SampleMvpCardTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.infullmobile.android.infullmvp.sample.sample_mvp_custom_view
+
+import com.infullmobile.android.infullmvp.basetest.InFullMvpCardBaseTest
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.SampleMvpCard
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.SampleMvpCardModel
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.SampleMvpCardPresenter
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.SampleMvpCardView
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.di.SampleMvpCardModule
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.di.SampleMvpCardScope
+import dagger.Provides
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnit
+import org.mockito.junit.MockitoRule
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.mockito.Mockito.`when` as whenDo
+
+@RunWith(RobolectricTestRunner::class)
+class SampleMvpCardTest
+    : InFullMvpCardBaseTest<
+        SampleMvpCard,
+        SampleMvpCardPresenter,
+        SampleMvpCardView>() {
+
+    @get:Rule val rule: MockitoRule = MockitoJUnit.rule()
+
+    @Mock lateinit var mockedPresenter: SampleMvpCardPresenter
+    private val temperature = 42
+
+    @Test
+    fun shouldShowTemperature() {
+        // given
+        val expectedText = String.format(testedView.temperatureString, temperature)
+
+        // when
+        testedView.displayTemperature(temperature)
+
+        // then
+        assertThat(testedCustomView.presentedView.temperatureTextView.text).isEqualTo(expectedText)
```

Comment:
`testedCustomView.presentedView.temperatureTextView.text` chain like this is a bit long. It could be shorter if Base test class had a `presentedView` val